File: utils/consts.py
from classes.test_bin import TestBinance
from classes.test_platforms import Platform, TestBybit, TestOKX


class IPlatform:
    name: str
    obj: Platform

    def __init__(self, name: str, obj: Platform) -> None:
        self.name = name
        self.obj = obj


platforms: list[IPlatform] = [
    IPlatform(name='binance', obj=TestBinance),
    IPlatform(name='bybit', obj=TestBybit),
    IPlatform(name='okx', obj=TestOKX),]

platforms_lst = ['binance', 'bybit', 'okx']
date_format = "%Y-%m-%d %H:%M:%S%z"
TZ = "Africa/Johannesburg"

File: packages/common/src/utils/consts.ts

import { TestBinance } from "@cmn/classes/test-binance";
import { TestBitget } from "@cmn/classes/test-bitget";
import { TestGateio } from "@cmn/classes/test-gateio";
import { TestKucoin } from "@cmn/classes/test-kucoin";
import { TestMexc } from "@cmn/classes/test-mexc";
import { TestPlatform, TestBybit, TestOKX } from "@cmn/classes/test-platforms";

export const test_platforms : {[key: string] : typeof TestPlatform} = {
    binance: TestBinance ,
    bybit: TestBybit ,
    okx: TestOKX ,
    kucoin: TestKucoin ,
    bitget: TestBitget,
    mexc: TestMexc,
};

 
